sdk/couchbase/src/results/get_result.rs

```diff
@@ -0,0 +1,31 @@
+use crate::error;
+use crate::transcoder::{DefaultTranscoder, Transcoder};
+use bytes::Bytes;
+use serde::de::DeserializeOwned;
+
+pub struct GetResult {
```

Comment:
Are we intentionally omitting CAS for now (and I suppose expiry too)? If so should we create a ticket/comment to get back to it?

Reply:
Yeah this change is just transcoding specific. There's already an issue for doing kv ops so it'll get covered in that.